lib/tapioca/dsl/helpers/active_record_column_type_helper.rb

```diff
@@ -102,7 +102,7 @@ def type_for_activerecord_value(column_type)
             "::Money"
           when ActiveRecord::Type::Integer
             "::Integer"
-          when ActiveRecord::Encryption::EncryptedAttributeType
+          when defined?(ActiveRecord::Encryption) && ActiveRecord::Encryption::EncryptedAttributeType
```

Comment:
I think if is dangerous to use `when` like this:
```ruby
irb> foo = nil
#=> nil

irb> case foo
irb> when defined?(Foo) && 42
irb>   puts "Oh no"
irb> when 42
irb>   puts "Here"
irb> end
Oh no
```
The current way only works because `defined?(Foo) && Foo` evaluates to `Foo` if it is defined, but it evaluates to `nil` if it is not.

I think what we want is:
```suggestion
          when ->(type) { defined?(ActiveRecord::Encryption) && type == ActiveRecord::Encryption::EncryptedAttributeType }
```

Reply:
Yes, I got confused with `type` and mistook it for the class itself, but it's actually an instance of the class we are checking against. Sorry for the misdirection.